### PR TITLE
New "Ground" injection

### PR DESCRIPTION
**Pull request by rolnico**
**Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**
https://github.com/powsybl/powsybl-core/issues/1131
But, in fact, ground disconnectors are not ignored but classified as disconnectors.

See for information first try in #2757 

**What kind of change does this PR introduce?**
Bug fix / feature

**What is the current behavior?**
No distinction is made between ground disconnectors and disconnectors in IIDM and no ground element exists

**What is the new behavior (if this is a feature change)?**
A new Ground identifiable is created and can be added next to a disconnector to modelise a "ground disconnector"

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [ ] No

**If yes, please check if the following requirements are fulfilled**

- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration steps are described in the following section

**What changes might users need to make in their application due to this PR? (migration steps)**

**Other information**:

### CODE REVIEW

File: iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractGroundTest.java [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * @author Nicolas Rol {@literal <nicolas.rol at rte-france.com>}
+ */
+public abstract class AbstractGroundTest {
+    private Network network;
+
+    @BeforeEach
+    public void initNetwork() {
```
Comment (So-Fras) [inline iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractGroundTest.java:31]: It looks like this network is a duplicate of the network created in `GroundSerDeTest`, at least if we omit the ground and the associated disconnector parts. If it is indeed the case, could it be possible to factorize this code in the iidm-test package?

**Review by So-Fras — CHANGES_REQUESTED** [review]
Thnak you for this PR.
I left 2 minor remarks.
In order to make the examples in tests more realistic, the disconnectors (or the bus in bus-breaker view) associated with the grounds could be linked with a line.

**Comment by So-Fras** [comment]
I am thinking about something: should we allow the ground creation in bus/breaker view? Will the associated disconnector be present on a bus/breaker network?

File: iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/NodeBreakerVoltageLevel.java [hunk 2]
```diff
@@ -464,29 +464,13 @@ public boolean isValid(UndirectedGraph<? extends TerminalExt, SwitchImpl> graph,
...
-                            break;
-
-                        case BUSBAR_SECTION:
-                            busbarSectionCount++;
-                            break;
-
-                        default:
-                            throw new IllegalStateException();
+                        }
+                        case LOAD, GENERATOR, BATTERY, SHUNT_COMPENSATOR, STATIC_VAR_COMPENSATOR, GROUND -> feederCount++;
```
Comment (flo-dup) [inline iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/NodeBreakerVoltageLevel.java:471]: adding the ground here means that the following cases are considered as a valid bus:
- a busbar with a ground
- a line and two grounds

I think we don't want that

**Review by flo-dup — CHANGES_REQUESTED** [review]

**Review by flo-dup — APPROVED** [review]

**Review by So-Fras — APPROVED** [review]

**Review by flo-dup — CHANGES_REQUESTED** [review]
Just realized we forgot to put it in TopologyVisitor!

**Review by flo-dup — APPROVED** [review]

**Pull request merged**